src/lib/inputs/InputRadio/InputRadio.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Radio as MuiRadio,
+  RadioProps,
+  FormControlLabel as MuiFormControlLabel,
+  FormControlLabelProps
+} from '@material-ui/core';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+
+export type InputRadioProps<T> = {
+  label: string;
+  labelPlacement?: FormControlLabelProps['labelPlacement'];
+  formikProps?: FormikProps<T>;
+} & RadioProps;
+
+export const InputRadio = ({
```

Comment:
@Carlo-Piantini this looks pretty well-typed, which is nice!

I have been doing React Functional Component types like this

```export const InputRadio: React.FC<InputRadioProps> = ...```

Reply:
This is resolved!